java/ql/src/Security/CWE/CWE-352/CsrfUnprotectedRequestType.qhelp

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE qhelp SYSTEM "qhelp.dtd">
+<qhelp>
+
+<overview>
+<p>When you set up a web server to receive a request from a client without any mechanism
+for verifying that it was intentionally sent, then it is vulnerable to a Cross Site Request
```

Comment:
`then it is vulnerable to a Cross Site Request`

It is not clear what `it` relates to here? Perhaps we should avoid using a pronoun, and using a specific subject instead 🤔